experimental/packages/otlp-exporter-base/src/platform/index.ts

```diff
@@ -14,4 +14,4 @@
  * limitations under the License.
  */
 export * from './node';
-export { OTLPExporterBrowserBase } from './browser';
+export * from './browser';
```

Comment:
For tree-shaking all exports should be explicit rather than implicit to avoid re-exporting internal values (even though I know this is the platform index.ts and not the root one)

Reply:
@MSNev I checked this in a few other packages' index.js files, I see wildcard import in all. Perhaps we could move to explicit imports at all places in a separate PR.

Reply:
Usage of the wildcard was historical from the early days. As we have not "yet" created other PR's to update the usage, I'd prefer to take the approach that if the package is touched (changed) that we change the usage at that point.

And as this is a "new" export, it should confirm to the explicit pattern.

While it would also be "nice" to change the `export * from './node'` to also be explicit as its existing then that one can be "delayed" to a later PR.

Reply:
@MSNev replaced all the wildcard exports to explicit ones.